templates/default/apache_ckan.conf.erb

```diff
@@ -1,6 +1,12 @@
 <VirtualHost 127.0.0.1:8000>
     ServerName <%= @app_url %>
     ServerAlias www.<%= @app_url %>
+
+    # Use canonical domain
+    RewriteEngine on
+    RewriteCond %{HTTP:X-Forwarded-Host} www\.([-_.a-z]*\.qld\.gov\.au)
```

Comment:
are you adding www. to our lower environments?
can we just check if its not if not www. beginning and redirect (unless domain ends in internal?)

Reply:
1) No, we would have to create new certificates to add `www` to lower environments. Do you want to do that?

2) I can't think of a situation where we should be applying this redirect to a domain that doesn't end in `qld.gov.au`.